src/loader.js

```diff
@@ -146,15 +153,89 @@
   };
 
   // 4. Registry Objects
+  // For now, registry objects are a work in progress that don't fully integrate into the rest of the code base
+
+  // 4.1.1 inlined in 4.2
+
+  // 4.2 - see https://github.com/ModuleLoader/es6-module-loader/pull/462#discussion-diff-50639828 for why it deviates from spec
+  function Registry() {
+    this._registry = {};
```

Comment:
I think we can set `this.registryMap` directly here. The use of `_loader` on Loader is done to mask the private class state on a public class. If this was public, I'd simply rename `this.registryMap` to `this._registryMap`. Ideally in modern environments we'd just use private Symbols here.